docs/src/docs/asciidoc/end-to-end-gradle-guide.adoc

```diff
@@ -28,13 +33,27 @@ The plugin requires that you https://www.graalvm.org/downloads/[install GraalVM]
   id("org.graalvm.buildtools.native") version "{gradle-plugin-version}"
 ----
 
-All plugin versions are listed https://github.com/graalvm/native-build-tools/releases[here]
+Replace `{maven-plugin-version}` with the latest released version.
+All plugin versions are listed https://github.com/graalvm/native-build-tools/releases[here].
+
+
+- Provide the application main class in _build.gradle_ / _build.gradle.kts_ as shown below.
```

Comment:
@dnestoro, is it a must-have step to enable the Gradle plugin? Also, is it still required to  declare an additional plugin repository in settings.gradle? 

```
 pluginManagement {
     repositories {
         mavenCentral()
         gradlePluginPortal()
     }
 }
```

Reply:
No, I was able to run few reproducers without that.